avaframe/runCom2AB.py

```diff
@@ -24,17 +24,15 @@
 else:
     cfg.read('avaframe/com2AB/com2ABCfg.ini')
 
-cfgPath = cfg['INPATH']
+cfgPath = com2AB.readABinputs(cfg['INPATH'])
 cfgsetup = cfg['ABSETUP']
 cfgFlags = cfg['FLAGS']
 
 #########################################
 # Load all input Parameters for logging #
 #########################################
 logFileName = os.path.join(cfgPath['saveOutPath'], "runCom2AB.log")
-if os.path.exists(logFileName):
-    os.remove(logFileName)
-logging.config.fileConfig(fname='avaframe/logging.conf', defaults={'logfilename': logFileName}, disable_existing_loggers=False)
+logging.config.fileConfig(fname='avaframe/in3Utils/logging.conf', defaults={'logfilename': logFileName}, disable_existing_loggers=False)
```

Comment:
What is the purpose of the 
disable_existing... flag? Do we need it?

Reply:
If you want to write the results in another logger, you need this flag. It is set to True as default. Since we only have one logger, it is fine to remove it, it was just in case